cirq/work/zeros_sampler.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import (List, Set, TYPE_CHECKING)
+
+import abc
+import numpy as np
+
+from cirq import work, study, protocols, ops, circuits
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class ZerosSampler(work.Sampler, metaclass=abc.ABCMeta):
+    """A dummy sampler for testing. Immediately returns zeroes."""
+
+    def run_sweep(
+            self,
+            program: 'cirq.Circuit',
+            params: study.Sweepable,
+            repetitions: int = 1,
+    ) -> List[study.TrialResult]:
+        """Samples circuit as if every measurement resulted in zero.
+
+        Args:
+            program: The circuit to sample from.
+            params: Parameters to run with the program.
+            repetitions: The number of times to sample.
+
+        Returns:
+            TrialResult list for this run; one for each possible parameter
+            resolver.
+        """
+        zero_measurement = np.zeros((repetitions, 1), dtype=np.int8)
```

Comment:
this doesn't work correctly for circuits with multi-qubit measurement gates

Reply:
I didn't know Cirq support that :)
Added support for multi-qubit measurement, PTAL.